frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -247,8 +251,46 @@ export function RenderedMarkdown({
     h6: CustomHeading,
     ...(overrideComponents || {}),
   }
-
-  const plugins = [remarkMathPlugin, remarkEmoji, remarkGfm]
+  const theme = useTheme()
+  const colors = theme.colors as typeof themeColors
+  const colorMapping = new Map(
+    Object.entries({
+      red: colors.red70,
+      blue: colors.blue70,
+      cyan: "#00C0F2",
```

Comment:
nit: maybe we move these colors into `theme.colors` as well as `cyan` & `teal`?

Reply:
Sure 👍 Now these are moved both to `baseTheme` and `darkTheme` and used accordingly by the Streamlit 👍